style.css

```diff
@@ -0,0 +1,1226 @@
+  line-height: 1.25;
+}
+
+.funding-article-title div h5 {
+  font-size: 11px;
+  font-weight: 500;
+  line-height: 1.5;
+}
+
+.funding-label {
+  padding: 16px;
+  border-bottom: 1px solid rgba(255, 68, 34, 0.25);
+}
+
+.funding-label a {
+  font-size: 13px;
+  color: #555;
+  line-height: 22px;
+  margin-bottom: 0.5em;
+}
+
+.label {
+  display: inline-block;
+  width: 16px;
+  margin-right: 20px;
+  font-size: 16px;
+}
+
+.funding-rounds-header a {
+  text-decoration: none;
+  color: #f42;
+  line-height: 18px;
+  margin-top: 11px;
+}
+
+.latest-deal-details h4 > a {
+  color: inherit;
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.article-details ul {
+  display: flex;
+}
+
+.article-details ul li {
+  margin-left: 0;
+  margin-right: 5px;
+}
+
+.article-details ul li > a,
+.article-details ul li > span {
+  font-size: 12px;
+  color: #aaa;
+}
+
+.funding a:nth-child(3) {
+  font-size: 12px;
+  font-weight: 400;
+  padding: 16px;
+}
+
+.funding-rounds {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+}
+
+.funding-rounds-header {
+  color: #f42;
+  margin-bottom: 24px;
+}
+
+.funding-rounds-header h2 {
+  font-style: sans-serif;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+.funding-rounds article {
+  min-height: 60px;
+  background: #fffaf8;
+  margin-bottom: 24px;
+}
+
+.funding-4 {
+  display: none;
+}
+
+.latest-deals header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.deal-1 {
+  background: url("images/latest-deals-1.jpg");
+}
+
+.deal-2 {
+  background: url("images/latest-deals-2.jpg");
+}
+
+.deal-3 {
+  background: url("images/latest-deals-3.jpg");
+}
+
+.deal-4 {
+  background: url("images/latest-deals-4.jpg");
+}
+
+.deal-5 {
+  background: url("images/latest-deals-5.jpg");
+}
+
+.deal-6 {
+  background: url("images/latest-deals-6.jpg");
+}
+
+.deal-7 {
+  background: url("images/latest-deals-7.jpg");
+}
+
+.deal-8 {
+  background: url("images/latest-deals-8.jpg");
+}
+
+del {
+  font-size: 12px;
+  color: #aaa;
+}
+
+footer {
+  border-top: 1px solid black;
+}
+
+.footer-links {
+  background: #131313;
+}
+
+.social-links {
+  margin: 0 auto;
+  padding-top: 3rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.site-links {
+  padding: 1.8rem 0 1.8rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.tar {
+  background: #333;
+  height: 45px;
+  width: 45px;
+  font-size: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  border: 2px solid #333;
+  border-radius: 50%;
+}
+
+.tar:hover {
+  background: #131313;
+}
+
+.legal-copyright {
+  background: #000;
+  color: #aaa;
+  padding: 3rem 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+@media screen and (max-width: 767px) {
+  .nav-bar-holder {
+    height: 3rem;
+  }
+
+  .secondary-menu {
+    display: none;
+  }
+
+  #main-menu-links {
+    display: none;
+  }
```

Comment:
- [ ] The same, here. Make the adjustments to avoid repetition of properties.

Reply:
Merged together. Thanks